packages/env/lib/env.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const fs = require( 'fs' );
+const dockerCompose = require( 'docker-compose' );
+const NodeGit = require( 'nodegit' );
```

Comment:
It'd be good to test that this works on Windows. `nodegit` is supposed to not require any native dependencies, Windows comes with nothing git-related installed (the GitHub app doesn't provide any global binaries or libraries, either), so it'd be a useful test bed.

Reply:
Yeah, it supports all major OSs, https://www.nodegit.org/.

Reply:
In release scripts, there is `simplegit` used:
https://github.com/WordPress/gutenberg/blob/d75119d2ebaafeeca53d9d665f2b3ed4ed31cf85/bin/commander.js#L12

Reply:
I was not entirely satisfied with `simple-git` in the release tool. at some point, I was wondering if just using the `git` CLI directly is better.

That said, there's some differences in the way this tool is written compared to the release tool (commands, logs...): both have similar requirements, they come up with different solutions (dependencies). It would be good to align.

Reply:
`nodegit` has a more robust API and doesn't require Git to be installed like `simple-git` does.

Reply:
> It would be good to align.

This is my biggest concern here. I don't care which one we use as long as we are consistent.